sdk/python/lib/pulumi/resource.py

```diff
@@ -374,6 +374,13 @@ class ResourceOptions:
     current package and should rarely be used.
     """
 
+    plugin_download_url: Optional[str]
```

Comment:
Similar feedback from #8698:
- Add `plugin_download_url ` to `InvokeOptions`
- Saving the URL on the resource and passing it along in `CallRequest` for `call`